test/framework/src/main/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -2441,6 +2442,11 @@ protected static RestClient createRestClient(
      */
     protected void setupSuiteScopeCluster() throws Exception {}
 
+    protected boolean autoManageVotingExclusions() {
+        // Temporary workaround until #98055 is tackled
```

Comment:
By just reading #98055 I am not sure I immediately understand the correlation to this PR.

Reply:
We should rely on the Shutdown API instead of orchestrating the cluster changes in the test framework